sonar-orchestrator/pom.xml

```diff
@@ -22,11 +22,6 @@
       <version>3.0.1</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>org.codehaus.sonar</groupId>
-      <artifactId>sonar-ws-client</artifactId>
-      <version>${sonarWsClient.version}</version>
```

Comment:
Definition of sonarWsClient.version should be removed, most probably

Reply:
@jacek-poreda-sonarsource did it 👍